### PR TITLE
Feature/60 team page

### PR DESCRIPTION
**Pull request by amoedoamorim**

### CODE REVIEW

File: localization/react-intl/src/app/components/team/InviteDialog.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "inviteDialog.invitationSent",
+    "description": "Sucess notification confirming that invitations were sent to users",
```
Comment (caiosba) [inline localization/react-intl/src/app/components/team/InviteDialog.json:4]: Typo: "sucess"

File: localization/react-intl/src/app/components/team/TeamDetails.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "id": "teamDetails.savedSuccessfully",
+    "defaultMessage": "Workspace details saved sucessfully"
```
Comment (caiosba) [inline localization/react-intl/src/app/components/team/TeamDetails.json:4]: Typo: sucessfully

File: src/app/styles/js/HeaderCard.js [hunk 3]
```diff
@@ -64,12 +52,6 @@ export const StyledHelper = styled.div`
   `}
 `;
 
-export const StyledProfileCard = styled.div`
-  margin-bottom: ${units(6)};
-  background-color: white;
-  box-shadow: ${boxShadow(1)};
-`;
-
 export const StyledContactInfo = styled.div`
```
Comment (caiosba) [inline src/app/styles/js/HeaderCard.js:55]: Do we still need this?

**Review by caiosba — CHANGES_REQUESTED** [review]

File: src/app/styles/js/HeaderCard.js (hunk 3 above)
Comment (amoedoamorim) [reply src/app/styles/js/HeaderCard.js:55]: Yes, this is still used in the `UserInfo` component. We'll get rid of it when we get there!

**Pull request merged**